lib/language.js

```diff
@@ -794,6 +794,22 @@ function init() {
       ,nb: '<ingen>'
       ,he: '<ללא>'
       }
+      'none' : {
+      cs: 'Žádný'
+      ,de: 'nichts'
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Expected an assignment or function call and instead saw an expression.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1241765613/issues/source?bid=2452379&fileBranchId=2935957#l799)